software/kernel/boards/ambient_module/Makefile

```diff
@@ -15,62 +13,44 @@ VCMD := echo \"pub static GIT_VERSION: &'static str = \\\"$$(git describe --alwa
 src/version.rs: FORCE
 	@bash -c "cmp -s <($(VCMD)) <(test -e $@ && cat $@) || $(VCMD) > $@"
 
-
-
 TOCKLOADER=tockloader
 
-JLINK=JLinkExe
-JLINK_OPTIONS+=-device ATSAM4LC8C -if swd -speed 1200 -AutoConnect 1
-JLINK_SCRIPTS_DIR =jtag
-
 # Where in the SAM4L flash to load the kernel with `tockloader`
-# KERNEL_ADDRESS=0x00000
+KERNEL_ADDRESS=0x10000
+
+BOOTLOADER_FILE = ../bootloader/justjump_bootloader.bin
```

Comment:
Should this, and the JTAG_FLAGS below, be in a common `signpost` Makefile? Feels like we're signing up for a lot of copy/paste (maybe a `Makefile.Signpost.SAM4L`)